### ISSUE TITLE
Sequence contains no matching element breaks all the webservices

### ISSUE THREAD

**Issue by MortenRa** [microsoft/AL#6247]
[TestWs.zip](https://github.com/microsoft/AL/files/5428982/TestWs.zip)
**Describe the bug**
When exposing a field in a query which has a "calcformula  = exist" it breaks all the webservices when query is exposed as webservice from version17.0  do however works in 16.5. 
One are allowed to publish a webservice but once you use the webservice the  error occurs in webclient: Sequence contains no matching element

if this is not allowed then the compiler should warn that you use a field with a CalcFormula = exist or CalcFormula = - Exist and should not be able to compile extension.

**To Reproduce**
Steps and to reproduce the behavior:
1. Go to web services and add your query, then click the webservice/odata URL

``` AL Code to reproduce the issue ```
AL project attaced [TestWs.zip](https://github.com/microsoft/AL/files/5428981/TestWs.zip)
use this query 

```
query 50100 "TEST WS"
{
    elements
    {
        dataitem(Item; "Item")
        {
            column(No; "No.")
            {
            }
            column(Assembly_BOM; "Assembly BOM")
            {
            }
        }
    }
}
```
**Expected behavior**
Should just work

Warn that you use a field with a calcformula = exist and should not be able to compile extension

**Screenshots**

**5. Versions:**

 - AL Language: 6.1..352568

 - Business Central: DK Business Central 17.0  (platform 17.0.16417.0 + program 17.0.16297.0

**Comment by dtkb** [comment]
Surely this should work; just making it a compile-time error would be the wrong solution.

**Comment by fvet** [comment]
This is also **breaking** our API's moving towards v17. 

**Expected behavior**
Should just work. It used to work on a BC 16.3.

@PooyaKharamesh 
Can you please change the label from CodeCop to **Bug**? Any fix expected in the BC 17.1?
